baseplate.go

```diff
@@ -242,8 +256,8 @@ func DecodeConfigYAML(reader io.ReadSeeker, serviceCfg interface{}) (Config, err
 // will also decode the config file at the path to set it up.  This can be used
 // to parse additional, service specific config values from the same config
 // file.
-func New(ctx context.Context, path string, serviceCfg interface{}) (context.Context, Baseplate, error) {
-	cfg, err := ParseConfig(path, serviceCfg)
+func New(ctx context.Context, args NewArgs) (context.Context, Baseplate, error) {
```

Comment:
Actually this doesn't work. The edgecontext implementation requires an already initialized secret store, but they don't have that before calling `New` 🤦

Reply:
:doh: